hack/common.sh

```diff
@@ -2,6 +2,10 @@
 
 # Location of the manifests file
 MANIFEST_LOC=bundle/
+COMMUNITY=${COMMUNITY:-false}
```

Comment:
I suggest moving this to the top of the file and adding a comment that explains why the default value is set to `false`. Then, handle the MANIFEST_LOC logic.